src/sign-in-xplat/README.md

```diff
@@ -10,26 +10,27 @@ products:
 - azure
 - azure-active-directory
 - ms-graph
```

Comment:
I think that's the right product name.  If it's not this, it's `dotnet-maui` -- I get "products" and "technologies" mixed up in the metadata. Maybe we just wait on this for content development.

```suggestion
- ms-graph
- dotnet-mobile
```